pkg/base1/cockpit.js

```diff
@@ -1280,6 +1280,10 @@ function basic_scope(cockpit, jquery) {
         return deferred.promise;
     };
 
+    cockpit.resolve = function resolve(result) {
```

Comment:
I'd say this should handle multiple arguments, so that one can write `cockpit.resolve(foo, bar, 1234)`.